tests/aws/services/lambda_/test_lambda.py

```diff
@@ -829,6 +831,55 @@ def test_lambda_url_echo_invoke(self, create_lambda_function, snapshot, aws_clie
         assert "Body" not in event
         assert event["isBase64Encoded"] is False
 
+    @markers.aws.validated
+    def test_lambda_url_headers_and_status(self, create_lambda_function, aws_client):
+        function_name = f"test-fnurl-echo-{short_uid()}"
+
+        create_lambda_function(
+            func_name=function_name,
+            zip_file=testutil.create_zip_file(TEST_LAMBDA_PYTHON_ECHO_JSON_BODY, get_content=True),
```

Comment:
nit: `handler_file=TEST_LAMBDA_PYTHON_ECHO_JSON_BODY` automatically configures `zip_file` and `handler`

Reply:
cool! i just copied it from the other tests :-) maybe in a follow up PR we can clean up the entire test suite to make sure it's consistent